views/js/qtiCreator/widgets/item/Widget.js

```diff
@@ -135,7 +135,9 @@ define([
                 reject(err);
             }
 
-            const xml = xmlNsHandler.restoreNs(xmlRenderer.render(this.element), this.element.getNamespaces(), true);
+            const xml = xmlNsHandler
+                .restoreNs(xmlRenderer.render(this.element), this.element.getNamespaces(), true)
+                .replace('&lcub;', '{');
```

Comment:
**question:** Is this change actually needed? The transformed braces would be part of the user's defined content, and it will display properly in the end. Unless this is an issue with the XML parser and the QTI validator?